docs/instruments/NDB_BVL_Instrument_UPLOADER_TEMPLATE.class.inc

```diff
@@ -149,13 +149,11 @@ class NDB_BVL_Instrument_TEST_NAME extends NDB_BVL_Instrument
             $file->processFiles();
         }
         //echo error messages
-        if(!empty($file->errorLog)){
-        while(list($fileType,$fileErrors)=each($file->errorLog)){
-            foreach($fileErrors AS $error){
-                 echo "<span style='color:red'><b>Upload Error</b> $fileType: $error</span><br>";
-            }
-         }
-        }
+        foreach($file->errorLog AS $fileType => $fileErrors){
+                foreach($fileErrors AS $error){
```

Comment:
indentation must be 4 spaces.

Reply:
Interesting! Why is this passing the php linter?

Reply:
I don't think php linter look at those.

Reply:
there is no docs in https://github.com/aces/Loris/blob/main/test/run-php-linter.sh#L63

Reply:
Oh, I understand. It's not a bad idea to add docs to this list.